src/table.rs

```diff
@@ -741,6 +1071,42 @@ where
         }
     }
 
+    /// An iterator producing the `usize` indices of all occupied buckets.
+    ///
+    /// The order in which the iterator yields indices is unspecified
+    /// and may change in the future.
```

Comment:
I think it would actually be fine to guarantee increasing order here, but the underlying `raw::FullBucketsIndices` is explicitly disclaiming that.

Reply:
I think it's fine to leave it unspecified for now unless there's a use case that clearly asks for it.